src/app/application.cpp

```diff
@@ -508,6 +509,7 @@ int Application::exec(const QStringList &params)
 
 #ifndef DISABLE_WEBUI
     m_webui = new WebUI;
+    SearchPluginManager::initInstance();
```

Comment:
SearchPluginManager already initializes it in its constructor, so this additional method isn't needed. Just call `new SearchPluginManager;` in place you want to initialize it (e.g. https://github.com/qbittorrent/qBittorrent/blob/master/src/app/application.cpp#L518).